src/storage/engine/mod.rs

```diff
@@ -126,6 +126,12 @@ pub trait Snapshot: Send + Clone {
     fn get_properties_cf(&self, _: CfName) -> Result<TablePropertiesCollection> {
         Err(Error::RocksDb("no user properties".to_owned()))
     }
+    fn lower_bound(&self) -> Option<&[u8]> {
```

Comment:
I think you can use inline for these functions. 
Btw, should add a comment for it.

Reply:
It will be inlined by the compiler, also i think the function name is quite clear.